examples/timer/timer_main.c

```diff
@@ -45,13 +45,15 @@
 #include <fcntl.h>
 #include <signal.h>
 #include <errno.h>
-
+#include <string.h>
 #include <nuttx/timers/timer.h>
 
 /****************************************************************************
  * Pre-processor Definitions
  ****************************************************************************/
 
+#define DEVNAME_SIZE 11
```

Comment:
Please change:
#define DEVNAME_SIZE 11 to 13
It is missing 1 byte to '\0' and also for some MCU with ten or more timers, /dev/timer10 will not fit

Reply:
done